README.md

```diff
@@ -4,34 +4,33 @@ The documentation content home for https://docs.quantum-computing.ibm.com. Refer
 
 # How tos
 
-## Pre-requisites to run tools locally
+## Prerequisites to run tools locally
 
-These tools will also run in CI. But, it can be convenient when iterating to run the tools locally.
-
-Currently, this workflow only works for IBM maintainers. We are prioritizing fixing this workflow so that anyone can run the tools locally, tracked by https://github.com/Qiskit/docs/issues/53.
+These tools will also run in CI, but it can be convenient when iterating to run the tools locally.
 
 First, install the below software:
 
 1. [Node.js](https://nodejs.org/en). If you expect to use JavaScript in other projects, consider using [NVM](https://github.com/nvm-sh/nvm). Otherwise, consider using [Homebrew](https://formulae.brew.sh/formula/node) or installing [Node.js directly](https://nodejs.org/en).
-2. [IBM Cloud command line](https://cloud.ibm.com/docs/cli?topic=cli-getting-started)
-3. [Docker](https://www.docker.com). You must also ensure that it is running.
+2. [Docker](https://www.docker.com). You must also ensure that it is running.
    * You can use [Colima](https://github.com/abiosoft/colima) or [Rancher Desktop](https://rancherdesktop.io). When installing Rancher Desktop, choose Moby/Dockerd as the engine, rather than nerdctl. To ensure it's running, open up the app "Rancher Desktop". 
 
-Set up IBM Cloud to access the Docker image:
-
-1. `ibmcloud plugin install cr`
-2. `ibmcloud cr region-set global`
-3. Make sure Docker is running, then `ibmcloud cr login`
-
-Finally, install the dependencies with:
+Then, install the dependencies with:
 
 ```bash
 npm install
 ```
 
 ## Preview the docs locally
 
-Run `./start` in your terminal, then open http://localhost:3000 in your browser.
+Run `./start` in your terminal, then open http://localhost:3000/start in your browser.
+
+The local preview does not include the top nav bar from docs.quantum-computing.ibm.com due to security considerations. So, you must directly navigate in the URL to the folder that you want:
```

Comment:
```suggestion
Note that you will reach a "page not found" screen, and this is expected. The local preview does not include the top nav bar from docs.quantum-computing.ibm.com due to security considerations. Therefore, you must directly navigate in the URL to the folder that you want:
```

Reply:
@Eric-Arellano I've added a note about expecting to reach a page-not-found screen -- what do you think?

Reply:
You'll only hit it if you reach `localhost:3000`. Otherwise you shouldn't have any problems. So I think it's fine to not mention 404s since we always tell you to go to a specific page?